integrations/kfasthttp/fasthttp.go

```diff
@@ -0,0 +1,122 @@
+package kfasthttp
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	//"github.com/fasthttp/router"
+	"github.com/keploy/go-sdk/keploy"
+	//routing "github.com/qiangxue/fasthttp-routing"
+	"github.com/valyala/fasthttp"
+	"github.com/valyala/fasthttp/fasthttpadaptor"
+	"go.keploy.io/server/pkg/models"
+	"go.uber.org/zap"
+	// "google.golang.org/genproto/googleapis/cloud/aiplatform/v1/schema/predict/params"
+)
+
+func captureResp(c *fasthttp.RequestCtx, next fasthttp.RequestHandler) models.HttpResp {
+	resBody := new(bytes.Buffer)
+	w := c.Response.BodyWriter()
+	mw := io.MultiWriter(w, resBody)
+	c.Response.WriteTo(mw)
+	writer := &bodyDumpResponseWriterFast{
+		Writer:   mw,
+		Response: c.Response,
+	}
+
+	header := http.Header{}
+	c.Response.Header.VisitAll(func(key, value []byte) {
+		k, v := string(key), string(value)
+		header[k] = []string{v}
+
+	})
+	next(c)
+	return models.HttpResp{
+		StatusCode: writer.Response.StatusCode(),
+		Header:     header,
+		Body:       resBody.String(),
+	}
+
+}
+
+func setContextValFast(c *fasthttp.RequestCtx, val interface{}) {
+	c.SetUserValue(string(keploy.KCTX), val)
+
+}
+func Fast(k *keploy.Keploy) func(fasthttp.RequestHandler) fasthttp.RequestHandler {
```

Comment:
@Pradhyuman-sharma you're already in a package called `kfasthttp`. Why did you call this method `fast`?

Reply:
ohh it is the middleware function which needs to be called at the time of integration like in chi router it is called as ChiMiddleware and so on in other routers.

Reply:
That needs to be corrected too. It's better to call it middleware or just mw is also fine

Reply:
I just made some change please take a look.